prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
+Estimate: 20 minutes
+Actual:   28 minutes
+
+"""
+
+word_to_count = {}
+text = input("Text: ")
+words = text.split()
+for word in words:
+    word_frequency = word_to_count.get(word, 0) #.get returns keyname/value doesnt exist yet
+    word_to_count[word] = word_frequency + 1
+
+words = list(word_to_count.keys())
+words.sort()
+print(word_to_count)
+
+max_length = max((len(word) for word in words))
+for word in words:
```

Comment:
Following what I said in previous comments. You can achieve similar results by using .items() to extract from the dictionary (instead of having to use a list).